test/lsp/utils/text_edit.vimspec

```diff
@@ -2,6 +2,7 @@ function! s:set_text(lines)
     % delete _
     put =a:lines
     execute 'normal ggdd'
+    execute 'write! /tmp/xyz'
```

Comment:
Is this line needed?

Reply:
That said there's maybe a better way to give a name to a buffer, without involving a disk write.

Reply:
@mattn I have changed to `file my-file` which has the benefit of non affecting the file system.
[7fce3b8](https://github.com/prabirshrestha/vim-lsp/pull/1552/commits/7fce3b8010b029843fdd7ccf53d3f55bd537e512)[7fce3b8](https://github.com/prabirshrestha/vim-lsp/pull/1552/commits/7fce3b8010b029843fdd7ccf53d3f55bd537e512)